deploy/acm-policies/cluster-admins/policy-generator-config.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: policy.open-cluster-management.io/v1
+kind: PolicyGenerator
+metadata:
+    name: cluster-admin-rolebinding
+policyDefaults:
+    namespace: openshift-rbac-policies
+    placement:
+        clusterSelectors:
+            hypershift.open-cluster-management.io/hosted-cluster: "true"
+    remediationAction: enforce
+    pruneObjectBehavior: "DeleteIfCreated"
+policies:
+    - name: cluster-admin
+      manifests:
+          - path: ./deploy/acm-policies/cluster-admins/cluster-admin-role-binding.yaml
```

Comment:
i changed it to ./deploy/osd-cluster-admin/03-cluster-admin.ClusterRoleBinding.yaml because the directory also contains config.yaml . would that be ok ?